src/main/java/org/opensearch/knn/common/launchcontrol/KNNLaunchControl.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.knn.common.launchcontrol;
+
+import org.opensearch.common.settings.Setting;
+import org.opensearch.knn.index.KNNSettings;
+
+import java.util.stream.Stream;
+
+public final class KNNLaunchControl {
```

Comment:
General terminology is feature flag, let use that in class name,

Reply:
LaunchControl was to not confuse it with existing Feature flag. I can change it back

Reply:
Lets use the semantics of Opensearch here it helps in better communication during discussions on github/any other maintainer as everyone understand what a feature flag is. It ensures consistency.

Otherwise at every place we have to write:
Enable/disable this Launch control, which is nothing but similar to feature flag.